### PR TITLE
[MOSIP-28823] Update registration-processor-camel-routes-lost-mz.xml

### PR DESCRIPTION
**Pull request by n-chandrasekhar**

### CODE REVIEW

File: sandbox/registration-processor-camel-routes-lost-mz.xml [hunk 1]
```diff
@@ -159,11 +159,11 @@
                 message="abis-handler-->abis-middleware lost route ${bodyAs(String)}" />
         <choice>
             <when>
-                <simple>${bodyAs(String)} contains '"address":"abis-middle-ware-bus-in"'</simple>
+                <jsonpath suppressExceptions="true">$.[?(@['tags']['ID_OBJECT-presentCity'] != 'Tubo' &amp;&amp; @['address'] == 'aabis-middle-ware-bus-in')]</jsonpath>
```
Comment (vishwa-vyom) [inline sandbox/registration-processor-camel-routes-lost-mz.xml]: spell check on aabis

File: sandbox/registration-processor-camel-routes-lost-mz.xml [hunk 2]
```diff
@@ -137,11 +137,11 @@
                 <to uri="eventbus:uin-generator-bus-in" />
```
Comment (vishwa-vyom) [inline sandbox/registration-processor-camel-routes-lost-mz.xml:137]: why is this missed ?

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: sandbox/registration-processor-camel-routes-lost-mz.xml (hunk 2 above)
Comment (n-chandrasekhar) [reply sandbox/registration-processor-camel-routes-lost-mz.xml:137]: Added condition

File: sandbox/registration-processor-camel-routes-lost-mz.xml (hunk 1 above)
Comment (n-chandrasekhar) [reply sandbox/registration-processor-camel-routes-lost-mz.xml]: Changed to abis-middle-ware-bus-in

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**